modules/ROOT/pages/batch-reference.adoc

```diff
@@ -0,0 +1,193 @@
+
+image::batch-job-fields-studio.png[alt=Batch Job fields in Studio,width=500px]
+
+The following table describes each field and includes the XML attribute for the field.
+
+[%header, cols="1a,1a,3a"]
+|===
+| Field Name | XML | Description
+
+| Name | `name` | Configurable name for a Batch Job component. The default name appends `Batch_Job` to the name of the application.
+
+| Max Failed Records | `maxFailedRecords` | Maximum number of records that can fail within a batch job instance before the execution of that instance stops. The default is `0`. Use `-1` for no limit. This setting takes precedence over any accept policy (`acceptPolicy`) or expression (`acceptExpression`) filter on a <<ref_batch_step_component, Batch Step component>> within the Batch Job component. When a batch job instance exceeds the `maxFailedRecords` value, the Batch Job component stops processing of the batch job instance in which the failure occurs, assigns the instance the status `FAILED_PROCESS_RECORDS`, and reports the number of failed records in the instance. When a batch job instance reaches this threshold, the console prints a message such as this one (edited for readability):
+
+----
+INFO  ... DefaultBatchEngine:
+ instance '6d85c380-f332-11ec-bb5f-147ddaaf4f97' of job 'Batch_Job_in_Mule_Flow'
+ has reached the max allowed number of failed records. Record will be added to
+ failed list and the instance will be removed from execution pool.
+...
+INFO  ... DefaultBatchEngine:
+ instance 6d85c380-f332-11ec-bb5f-147ddaaf4f97 of job Batch_Job_in_Mule_Flow
+ has been stopped. Instance status is FAILED_PROCESS_RECORDS
+----
+
+Note that parallel processing within a batch job instance makes it possible for the number of errors to exceed the configured maximum because it is possible for multiple records fail at the same time.
+
+| Scheduling Strategy | `schedulingStrategy` | When a Batch Job component triggers repeatedly, more than one batch job instance can be ready to execute at the same time. The instances can run sequentially based on their creation timestamp (the default) or according to a round-robin algorithm that attempts to maximize the use of available resources. This setting is appropriate only for jobs that have no side effects or dependencies on other records because the order of processing is not guaranteed.
+
+* `ORDERED_SQUENTIAL` (default): With this setting, batch job instances execute one at a time in the order established by their creation timestamp. An instance created earlier executes before an instance created at a later time and date.
+* `ROUND_ROBIN`: With this setting, all batch job instances execute based on a round-robin algorithm that assigns available resources. Order of execution is not predetermined. Batch instances can execute in any order. This option is viable only if you are certain that no execution of an instance is capable of producing a side effect on an execution of another instance. Best practices:
```

Comment:
```suggestion
* `ROUND_ROBIN`: With this setting, all batch job instances execute based on a round-robin algorithm that assigns available resources. Order of execution is not predetermined. Batch instances can execute in any order. This option is viable only if you are certain that no execution of an instance is capable of producing a side effect on an execution of another instance. When using this strategy, consider the following best practices:
```

Reply:
I like that sentence but think it's more scannable this way.